qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/offers/OfferDbConnector.groovy

```diff
@@ -115,10 +114,14 @@ class OfferDbConnector implements CreateOfferDataSource{
         }
     }
 
+    static String offerIdToString(OfferId id) {
+        return "${id.getProjectConservedPart()}_${id.getRandomPart()}_${id.getVersion()}"
+    }
+
```

Comment:
should we not use the converter here? Otherwise, we need to check multiple places if we change the format

Reply:
we should not, this belongs to the database representation and is a volatile detail that can change often. The Converter is part of the app domain and i dont expect it to change often, nor should it have knowledge about how data is represented in the database. SRP-

Reply:
I see so this is purely how we want the identifier to be stored in the database. Then it makes sense. Thanks for explaining it!